theano/tensor/nnet/conv.py

```diff
@@ -511,12 +592,16 @@ def __init__(self, imshp=None, kshp=None, nkern=None, bsize=None,
                 _logger.warn(warnstr, self.unroll_kern, self.nkern, new)
                 self.unroll_kern = new
 
-        self.outshp = ConvOp.getOutputShape(self.imshp_logical[1:],
-                                            self.kshp_logical, (dx, dy),
-                                            output_mode)
```

Comment:
While you're at it, could you change `ConvOp.getOutputShape` itself, so it calls `get_conv2d_output_shape`, and maybe emits a DeprecationWarning?